security_considerations.md

```diff
@@ -10,40 +10,55 @@ if an OEM or supplier deems it necessary. In this section, we discuss several of
 
-## Preventing rollback attacks in case of director compromise
+## Preventing rollback attacks in case of Director compromise
 
-On the Exceptional Operations page, we discussed how an OEM and / or its suppliers SHOULD use release counters in order to prevent attackers who compromise the director repository from executing rollback attacks. To further limit these attacks in this scenario, the OEM and / or its suppliers SHOULD also use the following recommendations.
+On the[Exceptional Operations](https://github.com/uptane/deployment-considerations/blob/master/exceptional_operations.md) page, we discussed how an OEM and / or its suppliers SHOULD use release counters in order to prevent rollback attacks should the Director repository be compromised. To further limit these attacks in this scenario, the OEM and / or its suppliers SHOULD also use the following recommendations.
 
-First, they SHOULD diligently remove obsolete images from new versions of targets metadata files uploaded to the image repository. This is done in order to prevent attackers who compromise the Director repository from being able to choose these obsolete images for installation. This method has the downside that it complicates updating vehicles that require intermediate updates in order to install the latest updates. For example, an ECU has previously installed image A, and C is the latest image it should install. However, the ECU should first install image B before it installs C, and B has already been removed from the targets metadata on the image repository in order to prevent / limit rollback attacks. Thus, the OEM and / or its suppliers SHOULD balance between the requirement to prevent / limit rollback attacks by removing obsolete images from the targets metadata, and the requirement to support updating ECUs that need to first install these obsolete images before updating to the latest images.
+First, they SHOULD diligently remove obsolete images from new versions of Targets metadata files uploaded to the Image repository. This can prevent attackers who compromise the Director repository from being able to choose these obsolete images for installation. This method has a downside in that it complicates the update process for vehicles that require an intermediate update step. For example, an ECU has previously installed image A, and C is the latest image it should install. However, the ECU should install image B before it installs C, and B has already been removed from the Targets metadata on the Image repository in order to prevent / limit rollback attacks. Thus, the OEM and / or its suppliers SHOULD balance between the requirement to remove obsolete images from the Targets metadata, and the need to support ECUs that require installation of these obsolete images before updating to the latest images.
```

Comment:
```suggestion
First, they SHOULD diligently remove obsolete images from new versions of Targets metadata files uploaded to the Image repository. This can prevent attackers who compromise the Director repository from being able to choose these obsolete images for installation. This method has a downside in that it complicates the update process for vehicles that require an intermediate update step. For example, an ECU has previously installed image A, and C is the latest image it should install. However, the ECU should install image B before it installs C, and B has already been removed from the Targets metadata on the Image repository in order to prevent or limit rollback attacks. Thus, the OEM and/or its suppliers SHOULD balance between the requirement to remove obsolete images from the Targets metadata, and the need to support ECUs that require installation of these obsolete images before updating to the latest images.
```